doc/src/Modify_contribute.rst

```diff
@@ -209,13 +209,13 @@ packages in the src directory for examples. If you are uncertain, please ask.
   A LaTeX citation is stored in a variable at the top of the file and a
   single line of code that references the variable is added to the
   constructor of the class.  Whenever a user invokes your feature from
```

Comment:
Mention this?  That is a style is enhanced over time with new features, that a cite for a new feature can be added, but the single line of code should be invoked only when the new feature is invoked.

Reply:
updated the paragraph accordingly